modules/nw-nwop-config-OpenShiftSDN.adoc

```diff
@@ -0,0 +1,36 @@
+// Deprecated
```

Comment:
If it's deprecated, why are you adding it?

Reply:
I didn't want to lose it. The file is a carry over from the previous PR. I'm going to delete it from the PR before merging.